mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.erb

```diff
@@ -1972,7 +1972,32 @@ func resourceSqlDatabaseInstanceUpdate(d *schema.ResourceData, meta interface{})
 		}
 	}
 
-	return resourceSqlDatabaseInstanceRead(d, meta)
+        // Check if timezone is updated.
```

Comment:
We'll need to remove `ForceNew: true` from `time_zone`, otherwise, Terraform will always delete/recreate the resource if `time_zone` is changed.